mechanics/combat/elemental-reactions/elemental-gauge-theory.md

```diff
@@ -14,127 +14,149 @@ If any words are unfamiliar, we have a entry on [common terms](../../../common-t
-| 2GU | 1.25GU |
+| 1GU | 0.5GU |
+| 2GU | 1GU |
 
-> An enemy affected by Amber’s Charged Shot has 2B Pyro. Using Kaeya’s E \(2B Cryo\) only removes 1.25GU Pyro because weak melt occurs when the trigger is Cryo.
+> An enemy affected by Amber’s Charged Shot has 1.6B Pyro. Using Kaeya’s E \(2B Cryo\) only removes 1GU Pyro because weak melt occurs when the trigger is Cryo.
 
-Strong amping elemental triggers have a 2.5x modifier.
+Strong amping elemental triggers have a 2x modifier.
 
 | Elemental Gauge of Source | After Unit Modifier |
 | :--- | :--- |
-| 1GU | 2.5GU |
-| 2GU | 5GU |
+| 1GU | 2GU |
+| 2GU | 4GU |
 
-> An enemy affected by Kaeya’s E has 2B Cryo. Using Diluc’s E \(1A\) removes 2.5GU worth of Cyro aura because strong melt occurs when the trigger is Pyro. This leaves us with 0GU Cryo as gauges cannot go below zero.
+> An enemy affected by Kaeya’s E has 1.6B Cryo. Using Diluc’s E \(1A\) removes 2GU worth of Cyro aura because strong melt occurs when the trigger is Pyro. This leaves us with 0GU Cryo as gauges cannot go below zero.
 
 ## Crystallize and Swirl
 
-Geo and Anemo can only be used as the trigger for crystallize and swirl reactions respectively. All Geo/Anemo triggers have a 0.625x modifier. Because Geo and Anemo cannot be applied as auras, they do not have an associated decay rate, so Geo and Anemo sources will only be refered to in GU.
+Geo and Anemo can only be used as the trigger for crystallize and swirl reactions respectively. All Geo/Anemo triggers have a 0.5x modifier. Because Geo and Anemo cannot be applied as auras, they do not have an associated decay rate, so Geo and Anemo sources will only be refered to in GU.
 
 | Elemental Gauge of Source | After Unit Modifier |
 | :--- | :--- |
-| 1GU | 0.625GU |
-| 2GU | 1.25GU |
-| 4GU | 2.5GU |
+| 1GU | 0.5GU |
+| 2GU | 1GU |
+| 4GU | 2GU |
 
-> Using a 1GU Geo trigger on a 1A Electro aura will result in Crystallize, subtracting 0.625GU and leaving 0.375A of Electro aura.
+> Using a 1GU Geo trigger on a 0.8A Electro aura will result in Crystallize, subtracting 0.5GU and leaving 0.3A of Electro aura.
 
-## Swirled Auras
+Unlike other transformative reactions which have an AOE, the AOE of swirl has a non-zero gauge, which shall be explored in the following
+section.
 
-Swirled Auras are auras applied to an enemy through the use of swirl spreading an element. Swirled Auras have their own unique decay notation that only applies to them, which can be seen in this table here:
+## Swirl Application
 
-| Anemo GU | Swirled Aura GU | Decay Notation | Decay Rate Conversion |
-| :--- | :--- | :--- | :--- |
-| 1GU | 2.25GU | 2.25S | 5.6s per S |
-| 2GU | 3.25GU | 3.25V | 4.5s per V |
+Swirl application occurs when an elemental source applied to an enemy through the use of swirl spreading an element. This may trigger additional reactions and or apply an aura following the same rules as any other elemental source. Notably, tax is applied to any swirl application that becomes an aura.
```

Comment:
@ANameOfSorts Minor issue with this sentence.

> Swirl application occurs when an elemental source applied to an enemy through the use of swirl spreading an element.

Should be something like "elemental source **is** applied..." or "aura **is** applied..." instead.